kaolin/datasets/shapenet.py

```diff
@@ -874,7 +874,7 @@ def get_name_index_mapping(self):
         for index, name in enumerate(self.names):
             self.name_to_index[name] = index
 
-    def get_tags_from_str(self, tags_str, inverse_order=True, forbidden_symbols=[" ", "/", "-", "\*"]):
+    def get_tags_from_str(self, tags_str, inverse_order=True, forbidden_symbols=[" ", "/", "-", "\\*"]):
```

Comment:
This is not related to the description of PR but I assume it's fixing some of the warning while using ShapeNet ?

Reply:
This warning during test run

```
=============================== warnings summary ===============================
--
1770 | kaolin/datasets/shapenet.py:877
1771 | /drone/src/kaolin/datasets/shapenet.py:877: DeprecationWarning: invalid escape sequence \*
1772 | def get_tags_from_str(self, tags_str, inverse_order=True, forbidden_symbols=[" ", "/", "-", "\*"]):
```